### PR TITLE
Change docker-compose to be "docker compose"

### PR DESCRIPTION
**Pull request by iperalta7**
### Desired Outcome

Migrate to docker compose v2.  Use `Docker compose` instead of `Docker-compose`

CyberArk internal issue ID: [CNJR-605](https://ca-il-jira.il.cyber-ark.com:8443/browse/CNJR-605)

### CODE REVIEW

File: ci/assets/Dockerfile.ci [hunk 1]
```diff
@@ -1,6 +1,6 @@
 FROM ruby:2.7-alpine
 
-RUN apk update && apk add --no-cache docker-cli git libffi-dev build-base bash jq curl
+RUN apk update && apk add --no-cache docker-cli docker-cli-compose git libffi-dev build-base bash jq curl
 
 # Install Docker-Compose
```
Comment (imheresamir) [inline ci/assets/Dockerfile.ci:5]: Can this section be removed now?

Comment (iperalta7) [reply ci/assets/Dockerfile.ci:5]: Would it actually be better to keep that line?
-  Essentially, instead remove the need to pip install docker compose. 
- The python package does not support compose v2 yet as compose v2 was built in Go and the python library still requires `docker-compose` hence still in compsoe v1 and does not recognize `docker compose` when the end-to-end tests are ran without the line. [pypi docker-compose](https://pypi.org/project/docker-compose/#:~:text=Docker%20Compose%20is%20a%20tool,up%20your%20application%20are%20configured.) 
- So, instead use alpine's package which has been maintained  to use compose v2 and it seems to still run.

Comment (imheresamir) [reply ci/assets/Dockerfile.ci:5]:
```
# Install Docker-Compose
RUN apk add --no-cache docker-cli python3 py3-pip && \
    apk add --no-cache --virtual .docker-compose-deps python3-dev libffi-dev openssl-dev gcc libc-dev make && \
    pip3 install docker-compose && \
    apk del .docker-compose-deps
```

I think we remove this section entirely, and add whichever packages out of `libffi-dev openssl-dev gcc libc-dev make` needed for `bundle install` to work correctly to the `RUN apk update && apk add` line above. Would that work?

**Review by imheresamir — APPROVED** [review]
LGTM!

**Pull request merged**